specification/cognitiveservices/data-plane/AnomalyDetector/stable/v1.1/MultivariateAnomalyDetector.json

```diff
@@ -0,0 +1,1187 @@
+{
```

Comment:
This json does not have "x-ms-parameterized-host"

Reply:
fixed.

Reply:
Syntax is wrong, you don't even have the 2 parameter in this JSON. Please fix.

Reply:
fixed.